_includes/cockroachcloud/planning-your-serverless-cluster.md

```diff
@@ -0,0 +1,11 @@
+### Resource usage
+
+All resource usage in CockroachCloud Serverless (beta) is measured in Request Units, or RUs. RUs represent the compute and I/O resources used by a query. All database operations cost a certain amount of RUs depending on the resources used. For example, a "small read" might cost 1 RU, and a "large read" such as a full table scan with indexes could cost 100 RUs. You can see how many request units your cluster has used on the [Cluster Overview](#view-cluster-overview) page.
```

Comment:
Nit: "Most resource usage in CockroachCloud Serverless (beta) is measured in Request Units" because storage usage is resource usage and it's not measured in request units.